src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -558,6 +576,41 @@ describe('Step Packages', () => {
         chosen = await screen.findByTestId('chosen-packages-count');
         expect(chosen).toHaveTextContent('1');
     });
+
+    test('should display empty available state on failed search', async () => {
+        const searchbox = screen.getAllByRole('textbox')[0]; // searching by id doesn't update the input ref
+
+        searchbox.click();
+
+        const getPackages = jest
+            .spyOn(api, 'getPackages')
+            .mockImplementation(() => Promise.resolve(mockPkgResultEmpty));
+
+        await searchForAvailablePackages(searchbox, 'asdf');
+        expect(getPackages).toHaveBeenCalledTimes(1);
+        screen.getByText('No packages found');
+    });
+
+    test('should display empty chosen state on failed search', async () => {
+        const searchboxAvailable = screen.getAllByRole('textbox')[0]; // searching by id doesn't update the input ref
+        const searchboxChosen = screen.getAllByRole('textbox')[1];
+
+        const getPackages = jest
+            .spyOn(api, 'getPackages')
+            .mockImplementation(() => Promise.resolve(mockPkgResult));
+
+        searchboxAvailable.click();
+        await searchForAvailablePackages(searchboxAvailable, 'test');
+        expect(getPackages).toHaveBeenCalledTimes(1);
+
+        screen.getByRole('button', { name: /Add all/ }).click();
+
+        searchboxChosen.click();
+        await searchForChosenPackages(searchboxChosen, 'asdf');
+        screen.getByText('No packages found');
+        // We need to clear this input in order to not have sideeffects on other tests
+        await searchForChosenPackages(searchboxChosen, '');
```

Comment:
Can we move this to the beforeEach?

Reply:
Specifically 612? or the whole block? beforeEach wouldn't work because it is also effecting other test groups/`describe`. I could put it in a afterEach but it is only used once and this side effect should hopefully disappear when I rewrite how we render the components as part of the eslint update